packages/dhis2/test/index.test.js

```diff
@@ -413,26 +394,63 @@ describe('upsert', () => {
       () =>
         execute(
           upsert(
-            'trackedEntityInstances',
+            'dataValueSets',
             {
-              ou: 'DiszpKrYNg8',
-              filter: ['w75KJ2mc4zz:Eq:John', 'zDhUuAYrxNC:Eq:Doe'],
+              orgUnit: 'DiszpKrYNg8',
             },
             {
               orgUnit: 'TSyzvBiovKh',
               trackedEntityType: 'nEenWmSyUEp',
-              attributes: [
-                {
-                  attribute: 'w75KJ2mc4zz',
-                  value: 'Qassim',
-                },
-              ],
             }
           )
         )(state),
       'Cannot upsert on Non-unique attribute. The operation found more than one records for your request.'
     );
   });
+
+  it('should make a post when new tracker is called', async () => {
```

Comment:
Good test, thank you. Small suggestion, can we rename the test to `should make a post only when new tracker is called` (I've added the word `only`)